src/client/src/apps/MainRoute/widgets/spotTile/SpotTileContainer.tsx

```diff
@@ -50,8 +50,10 @@ const mapDispatchToProps = (dispatch: Dispatch, ownProps: SpotTileContainerOwnPr
 const makeMapStateToProps = () => (state: GlobalState, ownProps: SpotTileContainerOwnProps) => ({
   pricingStatus: selectPricingStatus(state),
   executionStatus: selectExecutionStatus(state),
-  currencyPair: selectCurrencyPair(state, ownProps),
-  spotTileData: selectSpotTileData(state, ownProps),
+
+  // here 'ownProps.id' is an ID of the tile, but it's ID of the currency pair too (same thing for now)
+  currencyPair: selectCurrencyPair(state, ownProps.id),
```

Comment:
notice the change in the signature of `selectSpotTileData` selectors - it now receives simple ID instead of props of component - this makes it more reusable in the future